cluster-scope/base/operators.coreos.com/subscriptions/loki-operator/subscription.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: loki-operator
   namespace: openshift-operators-redhat
 spec:
-  channel: stable-5.6
+  channel: stable-5.8
```

Comment:
This is upgrading the loki operator on *all* clusters. Do we have it deployed anywhere other than the obs cluster? In particular, is it running on the prod cluster?

Reply:
@larsks @schwesig @bnshr  It's deployed on the `infra` cluster, and we are trying to deploy it on the `obs` cluster. I also realized that the Loki Operator is also closely related to the Logging Operator, which is configured to be `channel: stable`. Should we make Loki and Logging Operator both `channel: stable` so that we always have stable/supported logging, or increment versions?

Reply:
yes from my side.

Reply:
[larsks](https://massopencloud.slack.com/archives/C027TDE52TZ/p1707490673081909?thread_ts=1707489261.502159&cid=C027TDE52TZ)
I don't know; if stable will change versions (e.g., move from 5.7 to 5.8 automatically), that seems like it might introduce breaking changes. I would probably stick with stable-5.8, and only explicitly upgrade to a new minor release.

Reply:
call with @computate we think going with 5.7 on obs and then upgrade there with some smaller PRs would be the better solution

Reply:
@larsks we have updated the commit to only patch the loki-operator version on the obs cluster to `5.7`. Please review again.